client/my-sites/upgrades/checkout-thank-you/index.jsx

```diff
@@ -279,7 +320,8 @@ export default connect(
 	( state, props ) => {
 		return {
 			receipt: getReceiptById( state, props.receiptId ),
-			sitePlans: getPlansBySite( state, props.selectedSite )
+			sitePlans: getPlansBySite( state, props.selectedSite ),
+			user: getCurrentUser( state )
```

Comment:
We seem to only be using the date, so let's pass though that instead. It'll make our component's data needs clearer and easier to maintain.
```
getCurrentUserDate( state );
```